src/workloads/docs/Loader.yml

```diff
@@ -0,0 +1,79 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/product-perf"
+
+# More threads requires a larger connection pool.
+Clients:
+  Default:
+    QueryOptions:
+      maxPoolSize: 200
+
+Actors:
+  - Name: Loader
+    Type: Loader
+    Threads: 10
+    Phases:
+      - Repeat: 1
+        Database: cold
+        CollectionCount: 6
+        Threads: 1
+        DocumentCount: 200000
+        BatchSize: 100000
+        Document:
+          a: {^RandomString: { length: 100 }}
+      - Repeat: 1
+        Database: hot
+        CollectionCount: 6
+        Threads: 1
+        DocumentCount: 200000
+        BatchSize: 100000
+        Document:
+          a: {^RandomString: { length: 100 }}
+        FindOptions:
+          Hint: a_1 # Currently only support the index name.
+          Comment: "Phase 1 loader"
+      - Repeat: 1
+        Database: hot
+        CollectionCount: 6
+        Threads: 1
+        DocumentCount: 200000
+        BatchSize: 100000
+        Document:
+          a: {^RandomString: { length: 100 }}
+        FindOptions:
+          Hint: a_1 # Currently only support the index name.
+          Comment: "Phase 1 loader"
+      - Repeat: 1
+        Database: multithreaded
+        # Create 2 collections and populate with 5 threads / collection.
+        CollectionCount: 2
+        MultipleThreadsPerCollection: true
+        DocumentCount: 200000
+        BatchSize: 100000
+        Document:
+          a: {^RandomString: { length: 100 }}
+        FindOptions:
+          Hint: a_1 # Currently only support the index name.
+          Comment: "Phase 1 loader"
+  - Name: MultipleLoadThreadsPerCollection
```

Comment:
Didn't we just use 5 threads per collection in the last phase of the previous actor?

Reply:
yes but I also wanted to show a larger example.